doc/base/classes.xml

```diff
@@ -22390,7 +22390,19 @@
 			<argument index="0" name="rel_vec" type="Vector3">
 			</argument>
 			<description>
-				Move the body in the given direction, stopping if there is an obstacle. The returned vector is how much movement was remaining before being stopped.
+				Move the body in the given direction, stopping if there is an obstacle. If as a result of a movement there will be any collision then informations about this collision will be in returned dictionary.
+				Dictionary will contains those keys:
+				"position" - collision position
+				"normal" - collision normal
+				"local_shape" - id of this kinematic body shape that took part in a collision
+				"travel" -  traveled movement before being stopped
+				"remainder" - remaining movement before being stopped
+				"collider_id" - id of the collider, it can be used when dealing with [PhysicsServer]
+				"collider" - colliding body
+				"collider_shape_index" - index of the colliding shape, inside collider body "collider_metadata"
+				If the body did not intersect anything, then an empty dictionary (dir.empty()==true) is returned instead.
+
```

Comment:
All lines are treated as different paragraphs by the rst parser and by Godot's in-editor help, so this empty line will be an empty paragraph and should be removed. Same below.